lib/artist.rb

```diff
@@ -0,0 +1,50 @@
+class Artist
+
+  attr_accessor :name, :song, :musiclibrarycontroller, :musicimporter
+  extend Concerns::Findable
+
+  @@all = []
+
+  def initialize(name)
+    @name = name
+    @songs = []
+  end
+
+  def self.all
+    @@all
+  end
+
+  def self.destroy_all
+    @@all.clear
+  end
+
+  def save
+    @@all << self
+  end
+
+  def self.create(name)
+    self.new(name).tap do |artist|
+      artist.save
+      artist
+    end
+  end
+
+  def songs
+    @songs
+  end
```

Comment:
How could you write this another way in one line?

Reply:
Added `attr_reader :songs` instead of having a separate method for songs.